pkg/storage/bloom/v1/util.go

```diff
@@ -10,23 +9,8 @@ import (
 	"github.com/grafana/loki/v3/pkg/util/mempool"
 )
 
-type Version byte
-
-func (v Version) String() string {
-	return fmt.Sprintf("v%d", v)
-}
-
 const (
 	magicNumber = uint32(0xCA7CAFE5)
```

Comment:
Should this move to schema.go too? Feels a bit out of place here now.

Reply:
Fair point!